script/entrypoint

```diff
@@ -18,11 +18,14 @@ if [ -n "$GOOGLE_APPLICATION_CREDENTIALS" ] && which gcloud > /dev/null; then
 fi
 
 if [ "$#" = 0 ]; then
-    # Default to 20 workers because most of the tests are io-bound
-    exec pytest --black --flake8 --mypy-ignore-missing-imports --pydocstyle --numprocesses 20
+    # Default to 8 workers because most of the tests are io-bound
+    # When using pytest-xdist, total worker initialization time scales
+    # linearly to the number of workers. 8 workers seems to bring the best
+    # performance. See https://github.com/pytest-dev/pytest-xdist/issues/346
+    exec pytest --black --flake8 --mypy-ignore-missing-imports --pydocstyle -n 8
 elif [ "${1:0:1}" = - ]; then
     # First argument is a flag, assume intended executable is pytest
-    exec pytest "$@"
+    exec pytest "$@" -n 8
```

Comment:
What happens if the user invokes this and has already specified a `-n X` parameter? Will this error out? This may cause some confusion.

Reply:
Right. It doesn't error out, but it will always use `-n 8` even if someone wants to override it.